fixtures/dom/src/components/TestCase.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+const propTypes = {
+  children: React.PropTypes.node.isRequired,
+  title: React.PropTypes.node.isRequired,
+};
+
+class TestCase extends React.Component {
+
+  render() {
+    const { title, description, children } = this.props;
+
+    return (
+      <section className="test-case">
+        <h2 className="type-subheading">
+          <label>
+            <input type='checkbox' />
+            {' '} {title}
+          </label>
+        </h2>
+        <p>{description}</p>
+
+        {children}
+      </section>
+    );
+  }
+}
+
+TestCase.propTypes = propTypes;
+
+TestCase.Steps = ({ children }) => (
+  <div>
+    <h3>Steps to reproduce:</h3>
+    <ol>
+      {children}
+    </ol>
+  </div>
```

Comment:
Right now the fixtures can be tested with (almost) every version of React published. If we use functional components then it will break with versions `<14`.

It's probably safest to just use class components everywhere, but we can discuss limiting which versions we really need to support (probably not many)